doc/httpcli.md

```diff
@@ -0,0 +1,12 @@
+# HTTP endpoint
+
+Livepeer node exposes REST endpoint which is used by `livepeer_cli` for communication with node.
+By default, the CLI listens to localhost:7935. This can be adjusted with the -cliAddr `<interface>:<port>` flag.
+
+## Available endpoints:
+
+
+
+`/getLogLevel` returns currrent verbosity level in the body of response
+
+`/setLogLevel` sets verbosity current level. Level to set should be provided in body of the request, encoded as `application/x-www-form-urlencoded`. Parameter should be named `loglevel`.
```

Comment:
Should we give a CURL example of how to use the API, and maybe link to the valid log levels (or describe them here)?

Reply:
Added curl example